lrc/Pos/Lrc/Class.hs

```diff
@@ -0,0 +1,49 @@
+{-# LANGUAGE AllowAmbiguousTypes       #-}
+{-# LANGUAGE ExistentialQuantification #-}
```

Comment:
No, you need `ExistentialQuantification` separately. `RankNTypes` implies only `ExplicitForAll`, not `ExistentialQuantification` (which is not equal to `ExplicitForAll`).